<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleEditDlgClass</class>
 <widget class="QDialog" name="TitleEditDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TitleEditDlg</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>科目编号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="titleCode"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>科目性质:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="titleProperty">
      <item>
       <property name="text">
        <string>借</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>贷</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>科目说明:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="titleMemo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>TitleEditDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>TitleEditDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
